dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/Engine.scala

```diff
@@ -48,8 +48,8 @@ class Engine(context: EngineContext) {
   private val completionService =
     new ExecutorCompletionService[TaskSummary](executorService)
 
-  private val started: mutable.Set[TaskFingerprint] = mutable.Set()
-  private var held: List[ReachableByTask]           = List()
+  private val started: mutable.Set[(CfgNode, List[CfgNode])] = mutable.Set()
```

Comment:
Is this not just the `TaskFingerprint` class now? Why the use of tuple?

Reply:
True, let me get the task fingerprint back in. It's more descriptive.

Reply:
Once resolved and the merge conflict is sorted, then it's approved for me 👍